docs/OpenAPIExamples.rst

```diff
@@ -0,0 +1,344 @@
+
+Example call to ``get_file()``:
+
+.. literalinclude:: ../test/tutorial/scripts/api/get_file_api.py
+
+Example of the JSON returned by ``get_file()``:
+::
+
+    {
+      "describedBy": "https://schema.humancellatlas.org/type/file/7.0.2/sequence_file",
+      "schema_type": "file",
+      "file_core": {
+        "file_name": "SRR6546754_2.fastq.gz",
+        "file_format": "fastq.gz"
+      },
+      "read_index": "read2",
+      "insdc_run": [
+        "SRR6546754"
+      ],
+      "technical_replicate_group": "Rep_id_7031",
+      "provenance": {
+        "document_id": "39a93f75-0db3-4ee2-ab22-3eaa9932cf67",
+        "submission_date": "2019-01-30T11:15:21.403Z",
+        "update_date": "2019-02-19T17:17:10.540Z"
+      }
+    }
+
+
+``login``
+---------
+
+Configures and saves authentication credentials.
+
+Example call to ``login()``:
+
+.. literalinclude:: ../test/tutorial/scripts/api/login_api.py
+
+``logout``
+----------
+
+Clears authentication credentials previously configured with login.
+
+Example call to ``logout()``:
+
+.. literalinclude:: ../test/tutorial/scripts/api/logout_api.py
+
+``post_bundles_checkout``
+-------------------------
+
+Returns a ``checkout-job-id`` (e.g., ``4de1c603-fa8b-4c07-af37-06159e6951e0``). This
+``checkout-job-id`` can then be used with the ``get_bundles_checkout()`` method.
+
+Example call to ``post_bundles_checkout()``:
+
+.. literalinclude:: ../test/tutorial/scripts/api/post_bundles_checkout_api.py
+
+``post_search``
+---------------
+
+Find bundles by their ``bundle_fqid``, which is the bundle's UUID and version separated by a dot (.).
+
+For example, the bundle FQID ``fff807ba-bc98-4247-a560-49fb90c9675c.2019-08-01T200147.111027Z`` is
+a bundle with the UUID ``fff807ba-bc98-4247-a560-49fb90c9675c`` and the version number 
+``2019-08-01T200147.111027Z``.
+
+This method returns an FQID and URL for each matching bundle.
+
+Example call to ``post_search()``:
+
+.. literalinclude:: ../test/tutorial/scripts/api/post_search_api.py
+
+Example output:
+::
+
+    {
+        ...
+    },
+    {
+        "bundle_fqid": "fff807ba-bc98-4247-a560-49fb90c9675c.2019-08-01T200147.111027Z",
+        "bundle_url": "https://dss.data.humancellatlas.org/v1/bundles/fff807ba-bc98-4247-a560-49fb90c9675c?version=2019-08-01T200147.111027Z&replica=aws",
+        "search_score": null
+    },
+    {
+        ...
+    }
+
+
+``put_subscription, delete_subscription, get_subscription, get_subscriptions``
+------------------------------------------------------------------------------
+
+* ``get_subscriptions()``: Gets a list of users subscription.
+
+* ``put_subscription()``: Create a collection for the user given a replica and a call-back url.
+
+* ``get_subscription()``: Given the UUID of the subscription, show a subscription that the user created.
+
+* ``delete_subscription()``: Given a UUID and rpelica or the subscription, delete the subscription the user created.
```

Comment:
`Given a UUID and rpelica or the subscription`

->

`Given the UUID and replica of the subscription`

Reply:
Done. Thanks for catching these typos!